### PR TITLE
Tweak commcarehq repo config

### PR DESCRIPTION
**Pull request by dannyroberts**
and some other small changes

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -101,7 +101,7 @@ TRANSFER_SERVER = 'nginx'
 
 # Email setup
 # email settings: these ones are the custom hq ones
-{% if file_email or not EMAIL_LOGIN %}
+{% if EMAIL_LOGIN %}
```
Comment (snopoke) [inline src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2]: shouldn't this be `not EMAIL_LOGIN`?

File: src/commcare_cloud/environment/main.py [hunk 2]
```diff
@@ -267,6 +267,7 @@ def create_generated_yml(self):
             'dev_users': self.users_config.dev_users.to_json(),
             'authorized_keys_dir': '{}/'.format(self.paths.authorized_keys_dir),
             'known_hosts_file': self.paths.known_hosts,
+            'commcarehq_repository': self.fab_settings_config.code_repo,
```
Comment (snopoke) [inline src/commcare_cloud/environment/main.py]: where does this get used?

File: src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2 (hunk 1 above)
Comment (dannyroberts) [reply src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2]: Oops, yes definitely.

File: src/commcare_cloud/environment/main.py (hunk 2 above)
Comment (dannyroberts) [reply src/commcare_cloud/environment/main.py]: When ansible is first pulling down the code, it uses `commcarehq_repository` as the git endpoint. Not sure if it makes a real difference, but just seemed more correct.

**Pull request merged**